cleanlab_studio/studio/trustworthy_language_model.py

```diff
@@ -552,10 +588,13 @@ class TLMResponse(TypedDict):
         trustworthiness_score (float, optional): score between 0-1 corresponding to the trustworthiness of the response.
         A higher score indicates a higher confidence that the response is correct/trustworthy. The trustworthiness score
         is omitted if TLM is run with quality preset "base".
+
+        logs (dict, optional): a dictionary containing additional logs and metadata from the LLM call.
```

Comment:
```suggestion
        log (dict, optional): a dictionary containing additional logs and metadata returned from the LLM call only if the `log` key was specified in TLMOptions.
```

Reply:
how come your new type `TLMScoreResponse` is defined very differently to this `TLMResponse` object (in a different file, not as typedict, etc)?  

Couldn't `TLMScoreResponse` be defined here with a simple doctring: `Similar dict to TLMResponse, except without the response key.`

Reply:
I agree this is a little confusing.
The difference is that `TLMResponse` defines a typed dict - the definition specifies what keys/typing the dict will have. Whereas `TLMScoreResponse` is just a type alias - because the `get_trustworthiness_score` method can either return a float or a dictionary, hence the type alias:
```
TLMScoreResponse = Union[float, Dict[str, Any]]
```

> Couldn't TLMScoreResponse be defined here with a simple doctring: Similar dict to TLMResponse, except without the response key.

The reason why this would not work is because the `get_trustworthiness_score` by default returns just a float (not a dict like `TLMResponse`. 

I think renaming the `TLMScoreResponse` type alias would be the best way to keep this the least confusing.

Reply:
I agree that TLMScoreResponse correctly belongs in types

Reply:
but why isn't it instead defined as:

`TLMScoreResponse = Union[float, TLMScore]`

where `TLMScore` is defined as a TypedDict similar to `TLMResponse` and it has docstring like:

> Similar to TLMResponse dict, except without the response key.

?